### ISSUE TITLE
Upgrade to quiver ^0.29.0

### ISSUE THREAD

**Issue by chalin** [dart-lang/angular_components#263]
I'm currently [upgrading Angular example app component tests to use pageloader v3](https://github.com/dart-lang/site-webdev/pull/1585), but I ran into this issue:

> Resolving dependencies...
> ... because angular_components >=0.9.0-alpha+5 depends on quiver >=0.24.0 <0.29.0 and 
> every version of pageloader from git depends on quiver ^0.29.0, pageloader from git is incompatible with angular_components.

Could ACX be upgraded to use quiver ^0.29.0?

cc @kevmoo @kwalrath @mk13

**Comment by chalin** [comment]
A dependency override seems to do the trick.

**Comment by nshahan** [comment]
Yeah I can take a look at getting the dependency bumped or expanding the range to include v0.29.0. in an upcoming release. 

At first glance I noticed we will need to bump the dependency in the [date picker example](https://github.com/dart-lang/angular_components_example/blob/93755a4e3e07ea3121a0d5eeabec7be18bf06e38/example/material_datepicker_example/pubspec.yaml#L14), and the [observable package](https://github.com/dart-lang/observable/blob/1174dec23db3d4df0fd044d8d9abac059abbb2e3/pubspec.yaml#L12) as well.
